django/thunderstore/social/api/experimental/views/overwolf.py

```diff
@@ -82,6 +83,31 @@ def post(self, request: Request) -> HttpResponse:
         )
 
 
+class OwLogoutRequestBody(serializers.Serializer):
+    sessionid = serializers.CharField(label="Thunderstore session key")
+
+
+class OverwolfLogoutApiView(APIView):
+    """
+    Drop provided session from database.
+    """
+
+    permission_classes: Sequence[Type[BasePermission]] = [OauthSharedSecretPermission]
```

Comment:
Is the `OauthSharedSecretPermission` permission class necessary? Also `IsAuthenticated` would fit here nicely, which'd allow you to just yeet the `request.session` object this view got called with rather than parsing a request body with an arbitrary session ID.